components/modal/modal.html

```diff
@@ -0,0 +1,126 @@
+              attention of the user. It can be dismissed by either clicking on
+              the buttons or the backdrop
+            </p>
+
+            <div class="component-type">Checkbox Input</div>
+            <div class="component-example">
+              <div class="inp-container-row">
+                <button id="btn-modal" class="btn primary">Modal</button>
+              </div>
+            </div>
+            <div class="modal-backdrop"></div>
+            <div class="modal">
+              <div class="modal-container">
+                <div class="modal-header">
+                  <p>Modal Header</p>
+                </div>
+                <div class="modal-body">
+                  <p>Modal Body</p>
+                </div>
+                <div class="modal-footer">
+                  <div class="modal-actions">
+                    <button id="accept" class="btn success">Accept</button>
+                    <button id="decline" class="btn secondary">Decline</button>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="component-code">
+              <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%2866%2C66%2C66%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522modal-backdrop%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522modal%2522%253E%250A%2520%2520%2520%253Cdiv%2520class%253D%2522modal-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522modal-header%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253EModal%2520Header%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522modal-body%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253EModal%2520Body%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522modal-footer%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522modal-actions%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520id%253D%2522accept%2522%2520class%253D%2522btn%2520success%2522%253EAccept%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520id%253D%2522decline%2522%2520class%253D%2522btn%2520secondary%2522%253EDecline%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                style="
+                  width: 100%;
+                  height: 487px;
+                  border: 0;
+                  transform: scale(1);
+                  overflow: hidden;
+                "
```

Comment:
remove inline styling for iframes , add a seperate class for it in doc.css

Reply:
The height of iframe is different in class therefore i left it inline. Thanks for suggesting.